service.go

```
@@ -153,7 +153,7 @@ func (s *service) startRecording(channelID, teamID string) {
 	log.Printf("container successfully started")
 }
 
-func (s *service) stopRecording(channelID, teamID string) {
+func (s *service) stopRecording(channelID string) {
```

Huh, I thought gofmt didn't allow an unused parameter?

---

Well, the problem is that it's failing early since we have 2 `main` functions and missing all the other errors. Need to fix that.